tower-balance/src/lib.rs

```diff
@@ -106,6 +106,22 @@ where
         }
     }
 
+    /// Returns true iff there are ready services.
+    pub fn is_empty(&self) -> bool {
+        self.ready.is_empty()
+    }
+
+    /// Counts the number of services considered to be ready.
+    pub fn len(&self) -> usize {
```

Comment:
This isn't _really_ a collection structure, which `len` implies.

Maybe `num_ready()`?

Reply:
+1 to `num_ready()`